stingray/utils.py

```diff
@@ -33,7 +33,7 @@ def simon(message, **kwargs):
     warnings.warn("SIMON says: {0}".format(message), **kwargs)
 
 
-def rebin_data(x, y, dx_new, method='sum'):
+def rebin_data(x, y, yerr, dx_new, method='sum'):
```

Comment:
This also means that the return value should be optional. Let's discuss together, @evandromr and @dhuppenkothen (and whoever wants to chime in!)